components/desktop/remmina/Makefile

```diff
@@ -0,0 +1,60 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"). You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2020 Marco van Wieringen
+#
+
+BUILD_BITS= 64
+BUILD_STYLE= cmake
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		Remmina
+COMPONENT_VERSION=	1.4.7
```

Comment:
COMPONENT_NAME shall be all lower-case characters.

Reply:
That doesn't work as upstream names their dir Remmina if I change that the whole gmake build end ups throwing errors.

Reply:
Introduce COMPONENT_SRC_NAME= Remmina
and use this in the definition of COMPONENT_SRC.
For an example look at fonts/jetbrains-mono